Kite SDK/PSPrintSDK/KitePrintSDK-Info.plist

```diff
@@ -48,8 +48,6 @@
 	<key>UISupportedInterfaceOrientations</key>
 	<array>
 		<string>UIInterfaceOrientationPortrait</string>
-		<string>UIInterfaceOrientationLandscapeLeft</string>
-		<string>UIInterfaceOrientationLandscapeRight</string>
```

Comment:
That’s because you use a standard UINavigationController. If you use OLCustomNavigationController. It will be fine. But I have objections to that piece of code. Comments on that coming next.

> On Apr 9, 2015, at 3:39 PM, Deon Botha <notifications@github.com> wrote:
> 
> In Kite SDK/PSPrintSDK/KitePrintSDK-Info.plist <https://github.com/OceanLabs/iOS-Print-SDK/pull/72#discussion_r28067235>:
> 
> > @@ -48,8 +48,6 @@
> >  	<key>UISupportedInterfaceOrientations</key>
> >  	<array>
> >  		<string>UIInterfaceOrientationPortrait</string>
> > -		<string>UIInterfaceOrientationLandscapeLeft</string>
> > -		<string>UIInterfaceOrientationLandscapeRight</string>
> Ah my bad… This means a few screens rotate if you apply filters and I skip the OLProductHomeViewController, need to look into that…
> … <x-msg://4/#>
> —
> Reply to this email directly or view it on GitHub.
> 
>  <https://github.com/OceanLabs/iOS-Print-SDK/pull/72#discussion_r28066570> <https://github.com/OceanLabs/iOS-Print-SDK/pull/72/files#r28066570>
> —
> Reply to this email directly or view it on GitHub <https://github.com/OceanLabs/iOS-Print-SDK/pull/72/files#r28067235>.
>